install_config/aggregate_logging.adoc

```diff
@@ -323,9 +323,19 @@ ifdef::openshift-enterprise[]
 endif::openshift-enterprise[]
 |===
 +
-Running the deployer creates a deployer pod and prints its name. Wait until the
-pod is running. This can take up to a few minutes to retrieve the deployer image
-from the registry. You can watch its process with:
+Running the deployer creates a deployer pod and prints its name.
+
+. Wait until the pod is running.
+This can take up to a few minutes to retrieve the deployer image
```

Comment:
Strange construction. Perhaps "It may take several minutes for OpenShift to retrieve..."